### PR TITLE
PUB-2399 APIM Policy Files Refactoring

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-2399

### Change description ###

Remove ServiceNow ticket creation code from all policy files.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: infrastructure/resources/policy-files/Hearings/api-op-delete-hearing-policy.xml [hunk 1]
```diff
@@ -94,35 +94,8 @@
     <outbound>
         <base />
          <choose>
-            <when condition="@(context.Response.StatusCode == 401)">
+			 <when condition="@(context.Response.StatusCode == 401 || context.Response.StatusCode == 403)">
```
Comment (KianKwa) [inline infrastructure/resources/policy-files/Hearings/api-op-delete-hearing-policy.xml:97]: Small indentation issue here. Can it be changed?

File: infrastructure/resources/policy-files/CFT/api-op-request-direct-hearing-policy.xml [hunk 2]
```diff
@@ -94,35 +94,8 @@
     <outbound>
         <base />
          <choose>
-            <when condition="@(context.Response.StatusCode == 401)">
+			 <when condition="@(context.Response.StatusCode == 401 || context.Response.StatusCode == 403)">
```
[the remark this answers is not in the document]
Comment (junaidiqbalmoj) [reply infrastructure/resources/policy-files/CFT/api-op-request-direct-hearing-policy.xml]: fixed

File: infrastructure/resources/policy-files/Hearings/api-op-delete-hearing-policy.xml (hunk 1 above)
Comment (junaidiqbalmoj) [reply infrastructure/resources/policy-files/Hearings/api-op-delete-hearing-policy.xml]: fixed

File: infrastructure/resources/policy-files/Hearings/api-op-request-hearing-policy.xml [hunk 3]
```diff
@@ -94,35 +94,8 @@
 	<outbound>
         <base />
          <choose>
-            <when condition="@(context.Response.StatusCode == 401)">
+			 <when condition="@(context.Response.StatusCode == 401 || context.Response.StatusCode == 403)">
```
[the remark this answers is not in the document]
Comment (junaidiqbalmoj) [reply infrastructure/resources/policy-files/Hearings/api-op-request-hearing-policy.xml]: fixed

**Review by KianKwa — APPROVED** [review]

**Pull request merged**